src/components/general/Button.js

```diff
@@ -4,7 +4,11 @@ import styled from "styled-components";
 export default class Button extends Component {
   render() {
     const { type, children } = this.props;
-    return <StyledButton type={type}>{children}</StyledButton>;
+    return (
+      <StyledButton onClick={this.props.clickHandler} type={type}>
```

Comment:
I think that this should be `onClick` instead of `clickHandler` for consistency.

We should also add this to  destructuring on line 6, instead of grabbing it here. In other words, I think this should be:

```jsx
const { type, children, onClick } = this.props;
return (
  <StyledButton onClick={onClick} type={type}>
  // ...
```